components/org.wso2.carbon.identity.password.history/src/test/java/org/wso2/carbon/identity/password/history/handler/PasswordHistoryValidationHandlerTest.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.wso2.carbon.identity.password.history.handler;
+
+import org.mockito.Mock;
+import org.testng.Assert;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+import org.wso2.carbon.identity.core.handler.InitConfig;
+import org.wso2.carbon.identity.event.handler.AbstractEventHandler;
+import org.wso2.carbon.identity.password.history.constants.PasswordHistoryConstants;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.List;
+import java.util.ArrayList;
+
+public class PasswordHistoryValidationHandlerTest {
+
+    PasswordHistoryValidationHandler passwordHistoryValidationHandler;
+
+    @Mock
+    InitConfig initConfig;
+
+    @Mock
+    AbstractEventHandler abstractEventHandler;
+
+    @BeforeTest
+    public void setUp() throws IllegalAccessException, InstantiationException {
+        passwordHistoryValidationHandler = new PasswordHistoryValidationHandler();
+    }
+
+    @Test
+    public void testGetName() throws Exception {
+        Assert.assertEquals(passwordHistoryValidationHandler.getName(),
```

Comment:
Use static assert imports. Instead of  **Assert.assertEquals** -> use  **assertEquals**

Reply:
I fixed these issues